whois-api/src/main/java/net/ripe/db/whois/api/mail/dequeue/MessageParser.java

```diff
@@ -225,6 +226,12 @@ private void parseContents(@Nonnull final MessageParts messageParts, @Nonnull fi
         }
     }
 
+    private static void handleInvalidTypes(Part part) throws MessagingException {
+        if ("text".equals(part.getContentType())){
+            part.setHeader("Content-Type", "text/plain; charset=UTF-8");
```

Comment:
Be careful, you don't know what the character set is? Why default to UTF-8 ?

Reply:
At least for HTTP, if the type is "text" and no charset is defined, then the default is ASCII :
https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types#

Reply:
The RFC 6657 : https://datatracker.ietf.org/doc/html/rfc6657
updates the original rules, but still :
```
The default "charset" parameter value for "text/plain" is unchanged
   from [[RFC2046](https://datatracker.ietf.org/doc/html/rfc2046)] and remains as "US-ASCII".
```
Please review to confirm my understanding is correct, and fits your use case!

Reply:
yes, agree. by default is setting the charset to ISO-8859-1 so I don't need to specify nothing there